django_spanner/expressions.py

```diff
@@ -8,8 +8,16 @@
 
 
 def order_by(self, compiler, connection, **extra_context):
-    # In Django 3.1, this can be replaced with
-    # DatabaseFeatures.supports_order_by_nulls_modifier = False.
+    """Order expressions in the SQL query and generate a new query using
+    Spanner-specific templates.
+
+    TODO: In Django 3.1, this can be replaced with
+     DatabaseFeatures.supports_order_by_nulls_modifier = False.
+     Also, consider making this function a part of a class.
```

Comment:
This too, it can be a regular code comment but shouldn't be in the docstring.

Reply:
Same here.